src/user/user.py

```diff
@@ -21,23 +26,31 @@ def is_admin(self):
     @property
     def games_played(self):
         # Query game.players
-        return 0
+        return []
 
     @property
     def games_won(self):
         # Query Game.winners
-        return None
+        return []
 
     @property
     def level(self):
         return 1
 
     @property
+    def rating(self):
+        if not self.rank_data:
+            self.rank_data = Rating()
+            self.put()
+        return trueskill.Rating(
+            mu=self.rank_data.points, sigma=self.rank_data.elasticity)
+
+    @property
     def win_percentage(self):
         # self.games_won / self.games_played
         games_played = self.games_played
         if games_played:
-            return self.games_won / self.games_played
+            return len(self.games_won) / len(self.games_played)
```

Comment:
I think if we're going to go the route of having `games_won` be a list instead of a number, that it should *not* be an `@property`. Instead, be `get_games_won()` and `get_games_played`.

I was originally imagining them being:
```python
@property
def games_won(self):
    return Game.query(Game.winner_keys == self.key).count()
```

But if you want to actually fetch them, I think we should add a `get` prefix.